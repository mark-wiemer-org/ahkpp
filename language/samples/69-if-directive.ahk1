; [Issue #69](https://github.com/mark-wiemer-org/ahkpp/issues/69)
#If WinActive("ahk_class Notepad") or WinActive(MyWindowTitle) ;Comment
