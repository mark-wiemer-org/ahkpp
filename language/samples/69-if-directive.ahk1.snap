>; [Issue #69](https://github.com/mark-wiemer-org/ahkpp/issues/69)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>#If WinActive("ahk_class Notepad") or WinActive(MyWindowTitle) ;Comment
#^^^ source.ahk keyword.control.if.ahk
#   ^ source.ahk keyword.control.if.ahk
#    ^^^^^^^^^ source.ahk keyword.control.if.ahk support.function.ahk
#             ^ source.ahk keyword.control.if.ahk punctuation.bracket.ahk
#              ^ source.ahk keyword.control.if.ahk string.quoted.double.ahk punctuation.definition.string.begin.ahk
#               ^^^^^^^^^^^^^^^^^ source.ahk keyword.control.if.ahk string.quoted.double.ahk
#                                ^ source.ahk keyword.control.if.ahk string.quoted.double.ahk punctuation.definition.string.end.ahk
#                                 ^ source.ahk keyword.control.if.ahk punctuation.bracket.ahk
#                                  ^ source.ahk keyword.control.if.ahk
#                                   ^^ source.ahk keyword.control.if.ahk keyword.other.ahk
#                                     ^ source.ahk keyword.control.if.ahk
#                                      ^^^^^^^^^ source.ahk keyword.control.if.ahk support.function.ahk
#                                               ^ source.ahk keyword.control.if.ahk punctuation.bracket.ahk
#                                                ^^^^^^^^^^^^^ source.ahk keyword.control.if.ahk variable.def.ahk
#                                                             ^ source.ahk keyword.control.if.ahk punctuation.bracket.ahk
#                                                              ^^^^^^^^^ source.ahk keyword.control.if.ahk comment.line.semicolon.if.ahk
>